packages/app-store/hubspotothercalendar/api/callback.ts

```diff
@@ -55,5 +56,7 @@ export default async function handler(req: NextApiRequest, res: NextApiResponse)
   });
 
   const state = decodeOAuthState(req);
-  res.redirect(getSafeRedirectUrl(state?.returnTo) ?? "/apps/installed");
+  res.redirect(
+    getSafeRedirectUrl(state?.returnTo) ?? getInstalledAppPath({ variant: "other_calendar", slug: "hubspot" })
```

Comment:
This is to support redirecting to the correct category within the Installed Apps page once the app has been installed.